src/IECoreMaya/SceneShapeInterface.cpp

```diff
@@ -1470,3 +1484,81 @@ SceneInterface::Path SceneShapeInterface::fullPathName( std::string relativeName
 	return fullPath;
 }
 
+namespace
+{
+	bool parseConvertParams( std::map<std::string, std::string> &params, const MString &paramsStr )
```

Comment:
Is there a reason we're not using `IECore.ParameterParser` here? Just to avoid the python call I assume?